.docker/frontend

```diff
@@ -5,11 +5,11 @@ COPY /frontend/ /app/
 WORKDIR /app
 RUN yarn install
 
-VOLUME /app/node_modules
-VOLUME /app/.cache
-VOLUME /app/.parcel-cache
+#VOLUME /app/node_modules
+#VOLUME /app/.cache
+#VOLUME /app/.parcel-cache
```

Comment:
Why were these removed?

Reply:
one can create volumes  using docker-compose without declaring them in the Dockerfile. There was also an error with the paths, that referred to a frontend folder that did not exist in the dockerfile but did in the docker-compose. I commented this before with Pontus in Slack...Hope this makes it clearer.